src/rev_ai/models/asynchronous/transcript_type.py

```diff
@@ -0,0 +1,13 @@
+# -*- coding: utf-8 -*-
+"""Enum for transcript content types"""
+
+from enum import Enum
```

Comment:
@amikofalvy was there a reason we dint have this enum class but have one for captions